yarn-project/Dockerfile.prod

```diff
@@ -13,8 +13,8 @@ RUN ./scripts/version_packages.sh
 # Productionify. See comment in yarn-project-base/Dockerfile.
 RUN yarn workspaces focus @aztec/cli @aztec/aztec @aztec/aztec-faucet @aztec/aztec.js --production && yarn cache clean
 
-# We no longer need nargo.
-RUN rm -rf /usr/src/noir/noir-repo/target
+# We no longer need nargo. But keep acvm.
+RUN rm /usr/src/noir/noir-repo/target/release/nargo
```

Comment:
I understand that the `target` folder has a lot more stuff apart from the binaries. Maybe we [should delete everything _but_](https://unix.stackexchange.com/a/153863) `acvm`?